api/v1alpha1/nicclusterpolicy_types.go

```diff
@@ -79,6 +79,12 @@ type OFEDDriverSpec struct {
 	CertConfig *ConfigMapNameReference `json:"certConfig,omitempty"`
 	// Optional: Custom package repository configuration for OFED container
 	RepoConfig *ConfigMapNameReference `json:"repoConfig,omitempty"`
+	// TerminationGracePeriodSeconds specifies the length of time in seconds
+	// to wait before killing the OFED pod on termination
+	// +optional
+	// +kubebuilder:default:=300
+	// +kubebuilder:validation:Minimum:=0
+	TerminationGracePeriodSeconds int `json:"terminationGracePeriodSeconds,omitempty"`
```

Comment:
https://github.com/kubernetes/api/blob/92fe7a3ca2f4bd72b8f6faeab90fc75f4d1aab27/core/v1/types.go#L3173

lets make it the same as in k8s (i.e int64) WDYT ?

also do you see a benefit of having it a pointer vs non-pointer ?
for me, if we have default i think it fine to go with non-pointer type.

Reply:
> lets make it the same as in k8s (i.e int64) WDYT ?

Good catch, I agree

Reply:
updated to int64